src/memory/attribute.rs

```diff
@@ -563,51 +870,43 @@ impl<'topology> MemoryAttribute<'topology> {
         /// This attribute involves no initiator.
         ///
         /// Requires [`DiscoverySupport::numa_memory()`].
-        #[doc(alias = "HWLOC_MEMATTR_ID_CAPACITY")]
         HWLOC_MEMATTR_ID_CAPACITY -> capacity,
 
         /// Number of PUs in that locality (i.e. cpuset weight)
         ///
         /// Smaller locality is better. This attribute involves no initiator.
         ///
         /// Requires [`DiscoverySupport::pu_count()`].
-        #[doc(alias = "HWLOC_MEMATTR_ID_LOCALITY")]
         HWLOC_MEMATTR_ID_LOCALITY -> locality,
 
         /// Average bandwidth in MiB/s, as seen from the given initiator location
         ///
         /// This is the average bandwidth for read and write accesses. If the
         /// platform provides individual read and write bandwidths but no
         /// explicit average value, hwloc computes and returns the average.
-        #[doc(alias = "HWLOC_MEMATTR_ID_BANDWIDTH")]
         HWLOC_MEMATTR_ID_BANDWIDTH -> bandwidth,
 
         /// Read bandwidth in MiB/s, as seen from the given initiator location
         #[cfg(feature = "hwloc-2_8_0")]
-        #[doc(alias = "HWLOC_MEMATTR_ID_READ_BANDWIDTH")]
         HWLOC_MEMATTR_ID_READ_BANDWIDTH -> read_bandwidth,
 
         /// Write bandwidth in MiB/s, as seen from the given initiator location
         #[cfg(feature = "hwloc-2_8_0")]
-        #[doc(alias = "HWLOC_MEMATTR_ID_WRITE_BANDWIDTH")]
         HWLOC_MEMATTR_ID_WRITE_BANDWIDTH -> write_bandwidth,
 
         /// Average latency in nanoseconds, as seen from the given initiator location
         ///
         /// This is the average latency for read and write accesses. If the
         /// platform value provides individual read and write latencies but no
         /// explicit average, hwloc computes and returns the average.
-        #[doc(alias = "HWLOC_MEMATTR_ID_LATENCY")]
         HWLOC_MEMATTR_ID_LATENCY -> latency,
 
         /// Read latency in nanoseconds, as seen from the given initiator location
         #[cfg(feature = "hwloc-2_8_0")]
-        #[doc(alias = "HWLOC_MEMATTR_ID_READ_LATENCY")]
         HWLOC_MEMATTR_ID_READ_LATENCY -> read_latency,
 
         /// Write latency in nanoseconds, as seen from the given initiator location
         #[cfg(feature = "hwloc-2_8_0")]
-        #[doc(alias = "HWLOC_MEMATTR_ID_WRITE_LATENCY")]
         HWLOC_MEMATTR_ID_WRITE_LATENCY -> write_latency
```

Comment:
Unfortunately docalias is a collateral victim to addition of BUILTIN_ATTRIBUTES.